src/app/containers/Toggle/useToggle.jsx

```diff
@@ -1,23 +1,13 @@
 import { useContext } from 'react';
-import isToggleEnabled from './utils/isToggleEnabled';
-import { RequestContext } from '#contexts/RequestContext';
 import { ToggleContext } from '#contexts/ToggleContext';
 
-// This is temporary until we are fetching toggles from a remote endpoint
-const envMapping = {
-  local: 'local',
-  test: 'test',
-  stage: 'test',
-  live: 'live',
-};
-
 const useToggle = toggleName => {
-  const { env } = useContext(RequestContext);
-  const { toggleState } = useContext(ToggleContext);
+  const featureToggleObject = useContext(ToggleContext).toggleState[toggleName];
```

Comment:
Should we decide where custom hooks should live? Do they belong in the containers folder?